### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by tarmolov**
It's a simple version of `CONTRIBUTING.md` with basic advice and rules.

### CODE REVIEW

**Comment by SwinX** [comment]
IMO we have to add a useful note about issues - if someone fired issue which we are unable to reproduce and does not answer our questions for 1 week then this issue may be closed without further investigation.

**Comment by tarmolov** [comment]
:ship: ?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Pull requests and Code contributions
+
+* Tests must pass.
+* Follow our coding style (jscs and jshint will help you).
+* If you fix a bug, add a test.
+* If you can't fix a bug, file an [issue](https://github.com/gemini-testing/hermione/issues) with the steps to reproduce, the expected and the actual results.
+* if an issue desribes a problem which we are unable to reproduce and does not answer our questions for 1 week then this issue may be closed without further investigation.
```
Comment (SwinX) [inline CONTRIBUTING.md]: if an issue desribes a problem which we are unable to reproduce and `issue reporter` does not answer our questions..

**Comment by SwinX** [comment]
:ok:

**Comment by tarmolov** [comment]
Squashed commits and ready to merge.

**Comment by SwinX** [comment]
Sorry, I was not quite right quoting issue reporter - just wanted to point what exactly needs to be changed. Please remove quotes and we're ready to go.

**Comment by tarmolov** [comment]
Let's :rocket:

**Comment by SwinX** [comment]
Thank you!

**Pull request merged**